error: no template named 'unordered_map' in namespace 'std'

Compilation fails:
```
/wrkdirs/usr/ports/devel/ispc/work/ispc-1.24.0/src/opt/ImproveMemoryOps.cpp:713:17: error: no template named 'unordered_map' in namespace 'std'
    static std::unordered_map<std::string, GSInfo> replacementRules = {
           ~~~~~^
/wrkdirs/usr/ports/devel/ispc/work/ispc-1.24.0/src/opt/ImproveMemoryOps.cpp:931:17: error: no template named 'unordered_map' in namespace 'std'
    static std::unordered_map<std::string, int> checksDecompositionOfArgs = {
           ~~~~~^
/wrkdirs/usr/ports/devel/ispc/work/ispc-1.24.0/src/opt/ImproveMemoryOps.cpp:1169:17: error: no template named 'unordered_map' in namespace 'std'
    static std::unordered_map<std::string, GatherImpInfo *> replGatherToLoad = {
           ~~~~~^
/wrkdirs/usr/ports/devel/ispc/work/ispc-1.24.0/src/opt/ImproveMemoryOps.cpp:1215:17: error: no template named 'unordered_map' in namespace 'std'
    static std::unordered_map<std::string, ScatterImpInfo *> replScatterToStore = {
           ~~~~~^
/wrkdirs/usr/ports/devel/ispc/work/ispc-1.24.0/src/opt/ImproveMemoryOps.cpp:1399:17: error: no template named 'unordered_map' in namespace 'std'
    static std::unordered_map<std::string, Alignment> maskedStoreAlign = {
           ~~~~~^
/wrkdirs/usr/ports/devel/ispc/work/ispc-1.24.0/src/opt/ImproveMemoryOps.cpp:1481:17: error: no template named 'unordered_map' in namespace 'std'
    static std::unordered_map<std::string, Alignment> maskedLoadAlign = {
           ~~~~~^
6 errors generated.
```

This is because you didn't include <unordered_map> into ImproveMemoryOps.cpp

Clang
FreeBSD 13

--- a/src/opt/ImproveMemoryOps.cpp
+++ b/src/opt/ImproveMemoryOps.cpp
@@ -7,6 +7,8 @@
 #include "ImproveMemoryOps.h"
 #include "builtins-decl.h"
 
+#include <unordered_map>
+
 namespace ispc {
 
 using namespace builtin;

--- a/src/opt/ReplaceMaskedMemOps.cpp
+++ b/src/opt/ReplaceMaskedMemOps.cpp
@@ -10,6 +10,7 @@
 #include <llvm/Transforms/Utils/Local.h>
 
 #include <numeric>
+#include <unordered_map>
 
 namespace ispc {
 

--- a/src/opt/ReplacePseudoMemoryOps.cpp
+++ b/src/opt/ReplacePseudoMemoryOps.cpp
@@ -7,6 +7,8 @@
 #include "ReplacePseudoMemoryOps.h"
 #include "builtins-decl.h"
 
+#include <unordered_map>
+
 namespace ispc {
 
 using namespace builtin;

